src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -48,6 +48,8 @@ ContextAttribute::ContextAttribute()
   value                 = "";
   compoundValueP        = NULL;
   typeFromXmlAttribute  = "";
+  providingApplication  = "";
+  found                 = true;
```

Comment:
Just one thing.
Found means found locally or found remotely (SccFound) ?
Perhaps we should use a more descriptive name of the variable ...
For example: 'bool remote;', initialized to false and set to true if a CPr is found for the attribute.